### ISSUE TITLE
Rename "Copy Public Gateway URL" 

### ISSUE THREAD

**Issue by lidel** [ipfs-shipyard/ipfs-companion#857]
### TL;DR

When user wants to share link, they usually copy URL from address bar.
This brings UX issue when resource was loaded from local IPFS node running on user's machine.

### Problem

Sharing `http://` link to the canonical gateway and then upgrading it to IPFS if User Agent supports it would be ideal, but we don't have API for manipulating the URL copied from address bar and probably won't see one due to the potential abuse such API could bring.

### Existing solution 

The best we can do right now is to provide menu item for copying a shareable link to the IPFS resource at a public gateway.

The goal is to make it easier to share links to IPFS resources, however I don't believe the currently used label is descriptive enough:
https://github.com/ipfs-shipyard/ipfs-companion/blob/7575d09ee68d0d22f5da70711a267e8f8281fca5/add-on/_locales/en/messages.json#L91

>  ![2020-04-02--00-38-21](https://user-images.githubusercontent.com/157609/78193242-5b9bf100-747a-11ea-9eaa-362c49676511.png)
>
>  ![2020-04-02--00-38-51](https://user-images.githubusercontent.com/157609/78193249-6060a500-747a-11ea-9e5e-b48423da02d3.png)

### How to improve it?

Multiple folks seemed to be surprised this exists.  We will redesign Companions UI in the future, but perhaps we could make a small  change  sooner?

I imagine changing label + adding some icon could help in discovery.

Would something like `Copy link for sharing` be better?

Other suggestions are welcome. Please comment below.

**Comment by jessicaschilling** [comment]
"Copy shareable link"?
